### DISCUSSION TITLE
Run Workflows triggered by a GitHub Action

### DISCUSSION THREAD

**Discussion by jnodorp-jaconi** [community/community#27039] [Actions]
<p>Hi,</p>
<p>I have two workflows:</p>
<ol>
<li>A release workflow, creating a git tag / GitHub release for each commit merged into my main branch.</li>
<li>A publish workflow, building and pushing a docker image whenever a tag is created.</li>
</ol>
<p>The second workflow does not run, as it is triggered by a GitHub action.</p>
<p>Is there a way to force the second workflow to run?</p>
<p>Thanks!<br>
Julian</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I assume the issue is that the first workflow pushes the tag using <code>GITHUB_TOKEN</code>? The direct route would be to use a PAT for the Git push operation instead, in that case the second workflow should run.</p>
<p>You might also be able to use a <code>workflow_run</code> event to start the second workflow, but in that case you need to somehow figure out which new tag has been created, assuming the name matters, so this approach might be more complicated.</p>

**Comment by jnodorp-jaconi** [comment]
<aside class="quote no-group" data-username="airtower-luna" data-post="2" data-topic="187578">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>I assume the issue is that the first workflow pushes the tag using <code>GITHUB_TOKEN</code> ?</p>
</blockquote>
</aside>
<p>Exactly!</p>
<aside class="quote no-group" data-username="airtower-luna" data-post="2" data-topic="187578">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>You might also be able to use a <code>workflow_run</code> event to start the second workflow, but in that case you need to somehow figure out which new tag has been created, assuming the name matters, so this approach might be more complicated.</p>
</blockquote>
</aside>
<p>I use the docker meta action, so the whole point is having a tag event.</p>
<p>Thanks for the pointer though!</p>

**Comment by jnodorp-jaconi** [comment]
<aside class="quote no-group" data-username="airtower-luna" data-post="2" data-topic="187578">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>The direct route would be to use a PAT for the Git push operation instead, in that case the second workflow should run.</p>
</blockquote>
</aside>
<p>This is what I ended up doing. Thank you!</p>
<p>Edit: Anyone with a similar issue should probably read this as well: <a href="https://github.community/t/action-does-not-trigger-another-on-push-tag-action/17148" class="inline-onebox">Action does not trigger another on push tag action</a></p>
